starcli/__main__.py

```diff
@@ -89,6 +90,12 @@
     help="Search for trending repositories by username",
 )
 @click.option("--debug", is_flag=True, default=False, help="Turn on debugging mode")
+@click.option(
+    "--auth", 
+    type=str, 
+    default="", 
+    help="GitHub personal access token in the format 'username:token'. This can help with rate limits. See: https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token"
```

Comment:
Thanks! The sentence ”This can help with rate limits...” and after that can be removed

Reply:
The docs link can be added in the readme when needed